src/components/Navbar/useNavbar.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { useAuth } from "../../routing/auth";
+
+function useNavbar() {
+  const auth = useAuth();
+  const navigate = useNavigate();
+
+  const [anchorElNav, setAnchorElNav] = React.useState(null);
+  const [anchorElUser, setAnchorElUser] = React.useState(null);
```

Comment:
import useStete from react instead of React.useState

Reply:
Done